client/verta/verta/runtime.py

```diff
@@ -209,7 +204,7 @@ class context:
     """
     def __init__(self, validate: Optional[bool] = False):
         self._validate = validate
-        self._logs_dict = dict()
+        self._logs_dict = {}
```

Comment:
Since you mentioned preferring `dict()` elsewhere before, I think:

I actually prefer using `dict()`, myself, if we're only creating an empty dictionary. To me personally, `{}` looks confusing because I forget whether it's an empty `dict` or an empty `set`.

Reply:
That's funny.  I only changed it because I noticed that pylint has started complaining about it, but I prefer `dict()` for exactly that reason.  Swapped it back